pkg/ensurance/runtime/sandbox.go

```diff
@@ -0,0 +1,87 @@
+package runtime
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	"k8s.io/klog/v2"
+)
+
+type sandboxByCreated []*pb.PodSandbox
+
+func (a sandboxByCreated) Len() int      { return len(a) }
+func (a sandboxByCreated) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a sandboxByCreated) Less(i, j int) bool {
+	return a[i].CreatedAt > a[j].CreatedAt
+}
+
+// ListPodSandboxes sends a ListPodSandboxRequest to the server, and parses
+// the returned ListPodSandboxResponse.
+// copied from kubernetes-sigs/cri-tools/cmd/crictl/container.go
+func ListPodSandboxes(client pb.RuntimeServiceClient, opts ListOptions) ([]*pb.PodSandbox, error) {
```

Comment:
possible to import the package instead of copy code?

Reply:
I see that pkg/ensure/runtime/container.go do the same thing, and the out param is not same